services/api.py

```diff
@@ -809,6 +809,20 @@ def service_nodes_by_ancestors(service_node_ids):
                 bbox_geometry_filter = munigeo_api.build_bbox_filter(ref, val, 'geometry')
                 queryset = queryset.filter(Q(**bbox_filter) | Q(**bbox_geometry_filter))
 
+        if 'category' in filters:
+            services_and_service_nodes = filters.get('category', None).split(',')
+            service_ids = []
+            servicenode_ids = []
+            for ser_or_ser_node in services_and_service_nodes:
```

Comment:
`ser_or_ser_node` as a variable name hurts my eyes for some reason. Could you say eg. `category` instead?